dynamics-nav/SEEK-Function--File-.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "SEEK Function (File)"
+description: SEEK Function (File) Sets a file pointer to a new position in an ASCII or binary file.
```

Comment:
Too short per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.